tensorflow/contrib/layers/python/layers/feature_column.py

```diff
@@ -74,7 +74,7 @@
       columns=[department_column, bucketized_age_column],
       hash_bucket_size=1000)
 
-Example of building tf.learn model using FeatureColumns:
+Example of building `Estimator` model using FeatureColumns:
```

Comment:
Yes, I have been checked the grammar as follows:
```
$ grep -r "\`A Estimator\`" . | wc -l
      0
$ grep -r "\`Estimator\`" . | wc -l
      100
$ grep -r "\`Estimators\`" . | wc -l
      1
```

The "Estimator" is the most frequently used word among the above.